### PR TITLE
fix local dashboard data population command

### PR DESCRIPTION
**Pull request by czue**
Not sure how this snuck in here. I must have run it successfully locally and then accidentally added it.

### CODE REVIEW

**Review by ShubhamBansal1997 — APPROVED** [review]

File: custom/icds_reports/management/commands/populate_local_db_with_icds_dashboard_test_data.py [hunk 1]
```diff
@@ -61,5 +61,5 @@ def _rename_location(location_name_column, old_name, new_name):
     AwcLocation.objects.filter(**{location_name_column: old_name}).update(**{location_name_column: new_name})
     with get_cursor(AwcLocationLocal) as cursor:
         cursor.execute(
-            f"UPDATE awc_location_local SET {location_name_column}='{new_name}' where {location_name_column} = '{old_name}'b"
+            f"UPDATE awc_location_local SET {location_name_column}='{new_name}' where {location_name_column} = '{old_name}'"
```
[the remark this answers is not in the document]
Comment (czue) [reply custom/icds_reports/management/commands/populate_local_db_with_icds_dashboard_test_data.py:64]: ignoring

**Comment by ShubhamBansal1997** [comment]
One thing which is weird, that it's a important management command which we have to run to do a fresh install specially for dashboard team, so it occurs to me whenever I did a fresh install but why the same doesn't happen to any other dev?

**Comment by czue** [comment]
> One thing which is weird, that it's a important management command which we have to run to do a fresh install specially for dashboard team, so it occurs to me whenever I did a fresh install but why the same doesn't happen to any other dev?

If I had to guess I'd guess either they already had an env set up, never needed maps, or they didn't understand what caused the issues they faced but more just felt "my local environment doesn't work". I think @saiprakash-v likely ran into this since I know he had trouble with this command, but we never got to the bottom of it and he found a workaround.

**Comment by saiprakash-v** [comment]
I ran populate_db command with "test_commcare_ucr_citus" as db instead of "commcare_ucr_citus" here in django settings file. (https://github.com/dimagi/commcare-hq/blob/master/CITUSDB_SETUP.md#django-settings)

**Pull request merged**